<?xml version="1.0"?>
<launch>
	<node pkg="sensor_fusion" type="pcd_integrater" name="pcd_integrater" output="screen">
        <param name="FILE_PATH"      type="string" value="/home/amsl/PCD/SQ2/20180717/GROUND/" />
        <param name="INTEGRATE_PATH" type="string" value="/home/amsl/PCD/SQ2/20180717/Map/" />
        <param name="MAP_NAME"       type="string" value="ground_map.pcd"/>
    </node>
</launch>
